src/wifi/AdafruitIO_WINC1500.h

```diff
@@ -129,7 +131,8 @@ class AdafruitIO_WINC1500 : public AdafruitIO {
       _status = AIO_SSID_INVALID;
     } else {
       _disconnect();
-      WiFi.setPins(_winc_cs, _winc_irq, _winc_rst, _winc_en);
+      // Comment out line below for MKR1010
+      //WiFi.setPins(_winc_cs, _winc_irq, _winc_rst, _winc_en); //
```

Comment:
The call to `setPins` for the MK1010 should be within the `void AdafruitIO_MKR1010::_connect() {` call here: https://github.com/adafruit/Adafruit_IO_Arduino/pull/136/files#diff-06928cbc26ae2797288999bea44e4226R36

Please see the AirLift class for reference: https://github.com/adafruit/Adafruit_IO_Arduino/blob/master/src/wifi/AdafruitIO_AIRLIFT.h#L176